stdlib/math.jou

```diff
@@ -39,6 +40,9 @@ declare ceil(x: double) -> double
 declare floor(x: double) -> double
 # Returns the integral value that is nearest to x, with halfway cases rounded away from zero.
 declare round(x: double) -> double
```

Comment:
You could also add float versions of `ceil`, `floor`, `round` and `fabs`. (They just get `f` to end of name, e.g. the float version of `fabs` is `fabsf`.)

Reply:
Ok, I will add the float versions of these functions.